lib/f_service/result/success.rb

```diff
@@ -84,6 +84,30 @@ def then
         yield(*to_ary)
       end
 
+      # Returns itself to the given block.
+      # Use this to chain multiple service calls (since all services return Results).
+      # It will short circuit your service call chain if only consists of catches.
```

Comment:
I didn't understand the last phrase.

Reply:
I meant like this, could be clearer though

```ruby
CustomService.(*args) # supposed to be a Success
    .catch { ... } # won't run
    .catch { ... } # won't run
    .catch { ... } # won't run
```